server.coffee

```diff
@@ -111,15 +119,29 @@ loadApp = (logger, loadedConfig) ->
     app.get "#{ APP_ROOT }/v1/health-check", (req, res) ->
       res.send('OK\n')
 
-    port = process.env.PORT ? loadedConfig.get('server.port').get() ? 5000
-    app.listen(port)
-
-    logger.log('Server listening on port %d in %s mode', port, app.settings.env)
+    if loadedConfig.get('server.https.options').get() instanceof Object
+      https = require 'https'
+      fs = require 'fs'
+      httpsOptions = _.mapObject loadedConfig.get('server.https.options').get(), (v, k) ->
+        if _.isString(v)
```

Comment:
Most https certs are stored in files. The https server requires either the entire cert string or a buffer. With this logic it checks to see if the file exists then we load the contents of the file into a buffer. By default I added an ssl/ path to store the cert files to help prevent the issue you're talking about from happening. I don't believe any of the options that we have to consider would take a forward-slash as part of the option string, so if the file exists then loading its contents should be the correct thing to do.

Here are all of the available options that we'd have to consider:
https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener

Some additional documentation on express/connect/https
* https://docs.nodejitsu.com/articles/HTTP/servers/how-to-create-a-HTTPS-server
* http://expressjs.com/3x/api.html#app.listen

Note: The only real other option that I see is to replace default.yaml with a custom config module that returns an object. That way we could require 'fs' and allow end-users to add the options similar to the options in the nodejitsu link above and specify when the option should be a file buffer instead of a string. I went with the option that kept default.yaml to make things as backwards compatible as possible.